### PR TITLE
Update readme with details on sink

### PR DESCRIPTION
**Pull request by dibbles**
# Changes

Additions made to the readme to detail some of the resources created by the sink triggered by the webhook.  This is to aid others currently looking at playing with the extension who may be trying to have the webhook trigger their own pipeline.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: webhooks-extension/README.md [hunk 1]
```diff
@@ -51,20 +53,76 @@ Access the Dashboard through its ClusterIP Service by running `kubectl proxy`. A
...
+
+#### Image PipelineResource
+
+A PipelineResource of type image is created with:
+
+  - `url` set to ${REGISTRY}/${REPOSITORY-NAME}:${SHORT-COMMITID} where, REGISTRY is the value set when creating the webhook, other values are taken from the webhook payload.
+
+#### A PipelineRun
+
+A PipelineRun for your chosen pipeline; in the namespace specified when your webhook was created; the values assigned to parameters on the pipelinerun are taken from values provided when configuring the webhook or from the webhook payload itself.
```
Comment (a-roberts) [inline webhooks-extension/README.md]: Nit but I'd use commas instead of so many semicolons 😄

File: webhooks-extension/README.md [hunk 2]
```diff
@@ -51,20 +53,76 @@ Access the Dashboard through its ClusterIP Service by running `kubectl proxy`. A
...
+
+    resources:
+    - name: docker-image
+      resourceRef:
+        name: foo-docker-image-1563812630
+    - name: git-source
+      resourceRef:
+        name: bar-git-source-1563812630
+
+    serviceAccount: ${SERVICE-ACCOUNT}
```
Comment (a-roberts) [inline webhooks-extension/README.md]: Does this need to be mentioned here? Should we just omit this line?

File: webhooks-extension/README.md (hunk 1 above)
Comment (dibbles) [reply webhooks-extension/README.md]: changed to commas

File: webhooks-extension/README.md (hunk 2 above)
Comment (dibbles) [reply webhooks-extension/README.md]: leaving it in for completeness

**Comment by a-roberts** [comment]
Great, thanks @dibbles

/meow hats
/lgtm

**Pull request merged**